proxy/m2_actions.c

```diff
@@ -1186,6 +1186,101 @@ action_m2_container_propdel (struct req_args_s *args, struct json_object *jargs)
 }
 
 static enum http_rc_e
+_m2_container_snapshot(struct req_args_s *args, struct json_object *jargs)
+{
+	GError *err = NULL;
+	gchar **urlv = NULL;
+	gchar **urlv_snapshot = NULL;
+	char type[64] = {};
```

Comment:
There is `LIMIT_LENGTH_SRVTYPE` already defined (in `metautils/lib/metatypes.h`) for this purpose.